code/modules/antagonists/bloodsucker/objects/bloodsucker_crypt.dm

```diff
@@ -208,7 +208,7 @@
 	..()
 	unbuckle_mob(M)
 
-/obj/structure/bloodsucker/vassalrack/unbuckle_mob(mob/living/buckled_mob)//, force=FALSE)
+/obj/structure/bloodsucker/vassalrack/unbuckle_mob(mob/living/buckled_mob, force=FALSE)
```

Comment:
I'm not really sure why this was commented out

Reply:
i dont remember